### PR TITLE
dts: arm: nxp: rt: Add dedicated rt1064 dtsi 

### PR DESCRIPTION
**Pull request by loicpoulain**
Create rt1064 dtsi and add ocram1 node

Signed-off-by: Loic Poulain <loic.poulain@linaro.org>

### CODE REVIEW

File: dts/arm/nxp/nxp_rt1064.dtsi [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019, Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_rt.dtsi>
+
+&flexram0 {
+	ocram1: ocram@20280000 {
```
Comment (MaureenHelm) [inline dts/arm/nxp/nxp_rt1064.dtsi]: RT1060 needs the same change. The difference between RT1060 and RT1064 is that the latter adds internal flash.

@galak, should we handle this with separate `nxp_rt1060.dtsi` and `nxp_rt1064.dtsi` files, or combine into `nxp_rt106x.dtsi`? I'm leaning toward combined.

Comment (galak) [reply dts/arm/nxp/nxp_rt1064.dtsi]: What the difference between RT1060 & RT1064.  I'm probably same combinded, and at some point we can introduce an SoC set if needed.

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
I like the flexspi flash change, but I'd like to see the ocram added to rt1060 too. I think we could do this with another layer of dtsi includes:

`nxp_rt1060.dtsi`: includes `nxp_rt.dtsi` and adds the ocram
`nxp_rt1064.dtsi`: includes `nxp_rt1060.dtsi` and adds the internal flash

**Comment by loicpoulain** [comment]
> I like the flexspi flash change, but I'd like to see the ocram added to rt1060 too. I think we could do this with another layer of dtsi includes:
> 
> `nxp_rt1060.dtsi`: includes `nxp_rt.dtsi` and adds the ocram
> `nxp_rt1064.dtsi`: includes `nxp_rt1060.dtsi` and adds the internal flash

Done!

**Review by MaureenHelm — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**